README.md

```diff
@@ -20,6 +21,11 @@ Sfi.ru Distribution
   * Generate ssh key in container and add it to Github (https://github.com/settings/ssh)
   * Run `./import.sh` from within the container
 
+####Troubleshooting
+  * <i>Error in docker-compose up</i> client and server don't have same version... Update docker
+  * <i>Error in docker-compose installation</i> Permission denied. You can at first download and then replace in target folder step by step.
```

Comment:
This sentence is not clear. Maybe: "You can first save the docker-compose binary to some local directory, and then move it with sudo to `/usr/local/bin/`"